node/core/dispute-coordinator/src/real/initialized.rs

```diff
@@ -331,6 +381,8 @@ impl Initialized {
 		};
 
 		if backing_validators_per_candidate.is_empty() && disputes.is_empty() {
+			// Mark this block as scraped succesfully, still we did not get anything out of this block.
```

Comment:
Comment is misleading, there was nothing to be scraped in the block, right?

Reply:
Indeed. Will make it accurate sir!